src/ae/aens.js

```diff
@@ -200,7 +204,7 @@ async function query (name, opt = {}) {
     },
     revoke: async (options = {}) => this.aensRevoke(name, { ...opt, ...options }),
     extendTtl: async (nameTtl = NAME_TTL, options = {}) => {
-      if (!nameTtl || typeof nameTtl !== 'number' || nameTtl > NAME_TTL) throw new Error('Ttl must be an number and less then 180000 blocks')
+      if (!nameTtl || typeof nameTtl !== 'number' || nameTtl > NAME_TTL) { throw new IllegalArgumentError('Ttl must be an number and less then 180000 blocks') }
```

Comment:
```suggestion
      if (!nameTtl || typeof nameTtl !== 'number' || nameTtl > NAME_TTL) throw new IllegalArgumentError('Ttl must be an number and less then 180000 blocks')
```
avoid extra changes

Reply:
@davidyuk  Linter throws an error here. moving the statement to next line without braces